plugins/janus_sip.c

```diff
@@ -1077,6 +1093,11 @@ static void janus_sip_session_destroy(janus_sip_session *session) {
 		janus_refcount_decrease(&session->ref);
 }
 
+static void janus_sip_message_key_destroy(char *key) {
+	if (key)
+		g_free(key);
+}
```

Comment:
This is actually not needed: you can use `g_free` as a `GDestroyNotify` callback directly.

Reply:
ack